smart-scan/src/main/java/PreviousSeasons.java

```diff
@@ -130,170 +60,57 @@ public static int setPWP(String team, String sku) throws IOException {
-            currentDate = ((Date)formatter.parse(currentDateString));
-
-        }
-        catch (Exception e)
-        {}
-        URL url = new URL("http://api.vexdb.io/v1/get_events?season=" + eventThing.getPreviousSeason(false) + "&team=" + team);
-        //System.out.println(url);
-        InputStreamReader reader = new InputStreamReader(url.openStream());
-        JsonParser jsonParser = new JsonParser();
-        JsonArray results = (JsonArray) jsonParser.parse(reader).getAsJsonObject().get("result"); //idk what the hell this is
-
-        for(JsonElement result: results) {
-            JsonObject temp = result.getAsJsonObject();
-            String stuff = String.valueOf(temp.get("sku"));
-            stuff = stuff.replace("\"", "");
-            previousdates.add(stuff);
-
-        }
-        //System.out.println(previousdates);
-        for(int i = 0; i<previousdates.size(); i++) {
-            Event newEvent = new Event(previousdates.get(i));
-            newEvent.setSeason();
-            String time = newEvent.getDatetime();
-            time = time.replace("%3A", "");
-            time = time.substring(0, time.indexOf("T"));
-            //System.out.println(time);
-            try
             {
-                DateFormat formatter;
-                formatter = new SimpleDateFormat("yy-MM-dd");
-                previousDateList.add((Date)formatter.parse(time));
-
+                System.out.println("Error, getOldSKU handler - date formatter");
             }
-            catch (Exception e)
-            {}
 
         }
 
-        //System.out.println(previousDateList);
-        //System.out.println(currentDate);
+        
+        
         Date nearest = (getDateNearest(previousDateList, currentDate));
-        //System.out.println(nearest);
+        
         int indexSKUlookup = previousDateList.indexOf(nearest);
-        String oldSKU = (previousdates.get(indexSKUlookup));
-        URL url2 = new URL("http://api.vexdb.io/v1/get_rankings?team="+team + "&sku="+oldSKU);
-        //System.out.println(url2);
+        return(previousdates.get(indexSKUlookup));
+    }
+    public static double setPCCWM(String team, String sku, boolean twoyears) throws IOException {
+        String oldSKU = getOldSKU(team, sku, twoyears);
+        JsonParser jsonParser = new JsonParser();
+        URL url2 = new URL("https://api.vexdb.io/v1/get_rankings?team="+team + "&sku="+oldSKU);
+        
         InputStreamReader reader2 = new InputStreamReader(url2.openStream());
 
         try {
             JsonArray stuff2 = (JsonArray) jsonParser.parse(reader2).getAsJsonObject().get("result"); //idk what the hell this is
-            //System.out.println(stuff2);
-            return(stuff2.get(0).getAsJsonObject().get("ap").getAsInt());
+            
+            return(stuff2.get(0).getAsJsonObject().get("ccwm").getAsDouble());
 
         }
         catch (Exception e) {
-            return(0);
+            return(0.0);
```

Comment:
Why the hell does your return have parentheses... it's not a function it's an operator.